parm/config/gfs/config.ufs

```diff
@@ -106,16 +113,19 @@ case "${fv3_res}" in
         export tau=8.0
         export rf_cutoff=100.0
         export fv_sg_adj=3600
-        export WRITE_GROUP_GDAS=1
-        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=1
-        export WRITE_GROUP_GFS=1
-        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=1
         ;;
     "C96")
         if [[ "${DO_NEST:-NO}" == "YES" ]] ; then
+          if [[ "${RUN}" = *gdas ]]; then
```

Comment:
@aerorahul I am unsure of how nesting works.  Should there be `gdas`, `gfs`, and `nest` grids, threading, write tasks, etc?  Or only `gfs` and `nest`?